webdriver/tests/classic/set_window_rect/set.py

```diff
@@ -196,10 +203,17 @@ def test_width_height_floats(session):
     {"width": 200, "y": 200},
     {"height": 200, "y": 200},
 ])
-def test_no_change(session, rect):
+def test_partial_input(session, rect):
     original = session.window.rect
     response = set_window_rect(session, rect)
-    assert_success(response, original)
+    value = assert_success(response, session.window.rect)
+
+    assert value["width"] == rect.get("width", original["width"])
+    assert value["height"] == rect.get("height", original["height"])
+    # Wayland doesn't return correct coordinates after changing window position.
+    if not is_wayland():
+        assert value["x"] == rect.get("x", original["x"])
+        assert value["y"] == rect.get("y", original["y"])
```

Comment:
@sadym-chromium i assume that we run Chrome by default with XWAYLAND so that it can still set the position of the window? So we probably should keep what we have here and not explicitely check the `x` and `y` coordinates which can be different between browsers.